packages/relay/internal/signalling/server.go

```diff
@@ -44,7 +48,16 @@ func NewServer(config ServerConfig, app *fiber.App) *Server {
 	}
 	server.storage.setParticipants(config.Participants)
 	server.oldPeersCleaner = utils.SetIntervalTimer(time.Minute, server.storage.deleteOldPeers)
-	return &server
+
+	adminsNetworks, err := parseAdminsNetworks(server.config.AdminsRawNetworks)
+
+	if err != nil {
+		return nil, fmt.Errorf("can not parse admins networks, error - %v", err)
+	}
+
+	server.adminsNetworks = adminsNetworks
+
```

Comment:
May be it will better to create two different structs: first for unmarsual json config file and second with prepared config for server

Reply:
Fixed